lib/dart_pad_widget.dart

```diff
@@ -69,6 +60,14 @@ class DartPad extends StatefulWidget {
   String get iframeStyle {
     return "width:${width}px;height:${height}px;";
   }
+
+  Map<String, String> get sourceCodeFileMap {
+    return {
+      'main.dart': code,
+      if (testCode != null) 'test.dart': testCode!,
```

Comment:
Where is solutionCode used? I assume you just forgot to add it to this map?

Reply:
My bad, I couldn't seem to get solution code to do anything so I've just snipped it for now.